extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -42,6 +42,7 @@
 import org.apache.druid.indexing.common.TestUtils;
 import org.apache.druid.indexing.common.task.NoopTask;
 import org.apache.druid.indexing.common.task.Task;
+import org.apache.druid.indexing.kafka.IncrementalPublishingKafkaIndexTaskRunner;
```

Comment:
Maybe we should just rename the class to `KafkaIndexTaskRunner`, same for the rabbit stream one.

Reply:
Renamed